students/msemerni/hw-5/js/index.js

```diff
@@ -0,0 +1,121 @@
+//// Task 1
+//// Find the biggest number from a sequence and return it
+
+//// 2 numbers
+function getMaxNumber0(number1, number2) {
+  if (number1 > number2) {
+    return number1
+  }
+  return number2;
+}
+
+function getMaxNumber1(number1, number2) {
+  const maxNumber = number1 > number2 ? number1 : number2;
+  return maxNumber;
+}
+
+//// 3 numbers
+function getMaxNumber2(number1, number2, number3) {
+  const maxNumber = Math.max(...[number1, number2, number3]);
```

Comment:
Hm... conversation number to an array and vise versa 😄 
```suggestion
  const maxNumber = Math.max(number1, number2, number3);
```